lzy/channel-manager/src/main/java/ai/lzy/channelmanager/services/ChannelManagerPrivateService.java

```diff
@@ -76,7 +76,7 @@ public void create(LCMPS.ChannelCreateRequest request, StreamObserver<LCMPS.Chan
         }
 
         final String channelName = request.getChannelSpec().getChannelName();
-        String operationDescription = "Create channel " + channelName;
+        String operationDescription = "Create channel " + channelName + " of execution " + request.getExecutionId();
```

Comment:
I'd suggest to format logs in more strict and formal format (less humman readable)
i.e. `Create channel, channelName: {name}, executionId: {executionId}, ...`

Rationale: (example) if you want to find all actions (full history) for specific channel name (or channel id) you can just grep logs by `channelName: {name}` or `channelId: {id}` -- and you'll get all information about channel from all services